src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -818,6 +820,39 @@ def test_containerapp_managed_service_binding_e2e(self, resource_group):
             JMESPathCheck('length(properties.template.containers[0].env[?name==`AZURE_MYSQL_HOST`])', 1),
             JMESPathCheck('length(properties.template.containers[0].env[?name==`AZURE_POSTGRESQL_HOST`])', 1)
         ])
+        self.cmd('containerapp update -g {} -n {} --bind kafkaconfluent,bootstrap_server={},kafka_key={},kafka_secret={}'.format(
```

Comment:
please update description for --bind with supported service information

Reply:
unsure of whether help.py is where to do this but i'll look around. if you have a suggestion, please lmk. i'll update history.rst too.

Reply:
I think you need to update params.py to update the description for --bind/--unbind.
In addition, adding one more example in help.py is good for customer usage